src/sumOfOther.js

```diff
@@ -0,0 +1,18 @@
+module.exports = function sumOfOther(arr) {
+    if (Array.isArray(arr)) {
+        let array = [];
+        let sumArr = arr.reduce(function (sum, item) {
+            return sum + item
+        }, 0);
+        for (let i = 0; i < arr.length; i++) {
+            array.push(sumArr - arr[i]);
+        }
+        return array
+    }
+    else {
+        return {
+            error: "Not array"
+        }
+    }
+}
+
```

Comment:
Not bad solution! Just for your information, how it could be simplified:
`if (Array.isArray(arr)) {
        let sumArr = arr.reduce(function (sum, item) {
            return sum + item
        }, 0);

        return arr.map(item => sumArr - item);
    }`